MekHQ/src/mekhq/gui/adapter/PartsTableMouseAdapter.java

```diff
@@ -129,31 +130,25 @@ public void actionPerformed(ActionEvent action) {
                 }
             }
         } else if (command.contains("SET_QUALITY")) {
-            int q = -1;
-            boolean reverse = gui.getCampaign().getCampaignOptions().isReverseQualityNames();
+           boolean reverse = gui.getCampaign().getCampaignOptions().isReverseQualityNames();
+            // TODO : Duplicated in UnitTableMouseAdapter#actionPerformed
```

Comment:
Does this TODO need to be actioned before this is merged?

Reply:
I think we can remove both todos now actually, the code is only identical for some 10 lines and moving it into a function would be difficult. Plus the dialog boxes that are popped might wnat to be individually changed later?